render/src/bitmap.rs

```diff
@@ -158,3 +159,145 @@ impl BitmapFormat {
         }
     }
 }
+
+#[derive(Copy, Clone, Debug, Eq, PartialEq)]
+pub struct PixelRegion {
+    pub min_x: u32,
+    pub min_y: u32,
+    pub max_x: u32,
+    pub max_y: u32,
```

Comment:
Unimportant naming nit: Looks like `Rectangle` does `x_min` instead of `min_n`. Would be nice to use consistent naming convention here even if we end up merging them eventually.

Reply:
Renamed